docs/doc/30-reference/20-functions/60-conversion-functions/type-conversion.md

```diff
@@ -1,25 +1,143 @@
 ---
 title: Type Conversion
+title_includes: toBoolean, toDate, toDateTime, toTimestamp, toFloat32, toFloat64, toInt8, toInt16, toInt32, toInt64, toNull, toString, toUInt8, toUInt16, toUInt32, toUInt64
 ---
 
-to_int(8|16|32|64)
+Type conversion to target type. 
 
 ## Syntax
 
 ```sql
-to_int8(expr) — Results in the Int8 data type.
```

Comment:
The latest style is `to_intxx` after we merged #5126

Reply:
I noticed that, so I tried and found to_int8 can't work.

```sql 
MySQL [(none)]> select to_int8(1);
ERROR 1105 (HY000): Code: 2602, displayText = Unknown Function to_int8 (while in analyze select projection).
MySQL [(none)]> select toint8(1);
+-----------+
| toint8(1) |
+-----------+
|         1 |
+-----------+
```

Reply:
Sorry, maybe I don't get the latest version. I'll try it again.

Reply:
I am working on it, and some other improvements will be opened a PR soon.